overseerr-api.yml

```diff
@@ -1213,8 +1213,6 @@ components:
               type: string
             userToken:
               type: string
-            priority:
-              type: number
```

Comment:
So is this, technically.

Reply:
This was never used anywhere actually.  Should we not remove it?